R/mt_subset.R

```diff
@@ -205,3 +205,43 @@ mt_subset <- function(product = NULL,
                  out_dir = out_dir)
   }
 }
+
+corner_to_coord <- function (x) {
```

Comment:
This conversion could happen early in `mt_subset`, to avoid the shifted coordinates.